src/main/java/com.gigssandbox/command/CommandFromStringArray.java

```diff
@@ -0,0 +1,50 @@
+package com.gigssandbox.command;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CommandFromStringArray {
```

Comment:
Class name is not good. Firstly it imposes restrictions on implementation: you are omitted to reimplement it by using another storage for command. 

rename to Command